nifi-extension-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-client-service/pom.xml

```diff
@@ -83,23 +87,11 @@
             <artifactId>jackson-annotations</artifactId>
             <scope>provided</scope>
         </dependency>
-        <dependency>
-            <groupId>commons-io</groupId>
-            <artifactId>commons-io</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>org.apache.commons</groupId>
-            <artifactId>commons-compress</artifactId>
-        </dependency>
```

Comment:
Are these dependencies not used?

Reply:
IntelliJ reports that these dependencies are duplicated or not used here.

It looks like these are now inherited through the `nifi-elasticsearch-bundle`, which itself extends the `nifi-standard-shared-bom`, which provides these standard dependencies.

But this could be a mistake, I'm happy to add them back in if you think this might be an issue (I'm a little rusty on nested POMs) - things seem to be working OK when I run some tests though at least